dockstore-integration-testing/src/test/java/io/dockstore/common/Hoverfly.java

```diff
@@ -26,11 +26,14 @@
 import io.dockstore.webservice.core.Token;
 import io.dockstore.webservice.core.TokenType;
 import io.specto.hoverfly.junit.core.SimulationSource;
+import io.specto.hoverfly.junit.core.model.RequestFieldMatcher;
```

Comment:
Can't comment on the above but why is the LocalTrustStore increasing by 9500% in size?

Reply:
Hmm, @garyluu , should LocalTrustStore even be committed? Looks like it would be created in config.yml, or maybe that requires a preexisting file?

First trying by reverting commit.

Reply:
Tests did not pass going back to previous version, nor deleting it entirely. Restoring for now and will investigate further.

I'm assuming my JDK's cacerts is bigger, I guess.

Reply:
ping @garyluu

Reply:
So the LocalTrustStore that I had only contained the Hoverfly certs.  There a whole bunch of other certs necessary which is imported in the CI using:
`keytool -importkeystore -srckeystore $JAVA_HOME/lib/security/cacerts -destkeystore LocalTrustStore -srcstorepass changeit -deststorepass changeit`

Do you know which cert was needed that's not in either of them?

Reply:
Yeesh, I had unnecessarily run the keytool command locally and committed that.

When I reverted via IntelliJ something went wrong, the file was ended up corrupt. I correctly reverted and it is now working correctly.

Summary: will rollback changes to this file.